src/Ombi.Api.Emby/Models/Media/EmbyProviderids.cs

```diff
@@ -9,5 +9,19 @@ public class EmbyProviderids
         public string Tvdb { get; set; }
         public string Zap2It { get; set; }
         public string TvRage { get; set; }
+        
+        public bool Any()
```

Comment:
This correct? Any() will return true if they are all `NullOrEmpty`?

Maybe create a `BaseProviderIds` class?

```
public class BaseProviderIds
{
    public string Tmdb { get; set; }
    public string Imdb { get; set; }
    public string Tvdb { get; set; }

    public bool Any() => 
        !string.IsNullOrEmpty(Imdb) || !string.IsNullOrEmpty(Tmdb) || !string.IsNullOrEmpty(Tvdb);
}
```

Reply:
Oh dear, don't know where my head was when I did that (and not using this method). Sorry for the half-baked PR.
Also thought about a base class but was not sure how to package it across Emby and Jellyfin, I'll mull it over. Bare with me.